docs/examples/imageset-config-filter-catalog.yaml

```diff
@@ -1,33 +1,78 @@
 # This config demonstrates how to mirror a single operator. 
 
-# By specifying a minVersion, that version and every version to the
-# specified channel's latest (HEAD) version will be mirrored. 
-
 # If a package is specified with no minVersion/maxVersion or channel, each channel's latest
 # version in the specified package will be mirrored.
 
-# If the package is specified with a minVersion and maxVersion, all bundles within that range
-# will be mirrored.
-# WARNING: Any specified channel information will override this setting. Setting versions by
-# package or channel are mutually exclusive.
+# IMPORTANT: This is only true if metadata for this package in unestablished.
+# Minimum bundle versions are managed in the metadata and the latest differential bundles will be included additively.
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+mirror:
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.10
+      packages:
+        - name: elasticsearch-operator
 ---
+# If a package is specified with a channel with no minVersion/maxVersion, only the channel's latest
+# version will be mirrored.
+
+# IMPORTANT: This is only true if metadata for this package in unestablished.
```

Comment:
I think repetition in multiple examples is good, but multiple times in same file is excessive imo